include/uacpi/acpi.h

```diff
@@ -1003,6 +1003,131 @@ UACPI_PACKED(struct acpi_ecdt {
 })
 UACPI_EXPECT_SIZEOF(struct acpi_ecdt, 65);
 
+// Constants for acpi_dbg2_device_info->port_type.
```

Comment:
The "Constants for" prefix can be removed in most of these, other places in the file don't have it.

I guess places where it's conditional like `acpi_dbg2_serial_subtype ` do benefit from it though.